### PR TITLE
CANS-571. Create Back End for Supervisor Dashboard

### PR DESCRIPTION
**Pull request by denys-davydov**
## Description
https://osi-cwds.atlassian.net/browse/CANS-571

New endpoint added *GET /staff/subordinates* that returns all subordinates with assessment statistics for a logged in user who has a supervisor privileges.

## Tests
- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [x] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [ ] My code adds no new issues to Code Climate
- [x] I ran all the linters and static analyzers for the project (SonarQube, eslint, rubocop, etc).
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: config/cans-api.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@ cmsDataSourceFactory:
   minIdleTime: 1 minute
   properties:
     hibernate.dialect: org.hibernate.dialect.DB2Dialect
-    hibernate.show_sql: false
+    hibernate.show_sql: true
```
Comment (vladimirPe) [inline config/cans-api.yml]:
```suggestion
    hibernate.show_sql: false
```

File: src/main/java/gov/ca/cwds/cans/service/StaffService.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        staffList.stream().map(StaffBySupervisor::getRacfId).collect(Collectors.toSet());
+    if (staffRacfIds.isEmpty()) {
+      return Collections.emptyList();
+    }
+    final Map<String, Statistics> statisticsMap =
+        statisticsService.getStaffStatistics(staffRacfIds);
+    final Collection<StaffStatisticsDto> results = new ArrayList<>(staffRacfIds.size());
+    for (StaffBySupervisor staff : staffList) {
+      final Statistics statistics = statisticsMap.get(staff.getRacfId());
+      final StaffStatisticsDto statisticsDto = staffStatisticMapper.toDto(staff, statistics);
```
Comment (vladimirPe) [inline src/main/java/gov/ca/cwds/cans/service/StaffService.java]: what the purpose of this final modifier?

Comment (denys-davydov) [reply src/main/java/gov/ca/cwds/cans/service/StaffService.java]: By default, I declare all the variables as final unless it is planned to change a it's reference.

**Comment by denys-davydov** [comment]
Retest this please

**Pull request merged**